torch/onnx/symbolic_opset9.py

```diff
@@ -1900,3 +1900,12 @@ def multinomial(g, input, num_samples, replacement=False, generator=None):
     return g.op("Multinomial", log_input,
                 dtype_i=sym_help.cast_pytorch_to_onnx['Long'],
                 sample_size_i=num_samples)
+
+
+def remainder(g, input, other):
+    div = g.op("Div", input, other)
+    if input.type().scalarType() != 'Int':
+        div = g.op("Floor", div)
+    quo = g.op("Mul", div, other)
+    dtype = input.type().scalarType()
+    return g.op("Cast", g.op("Sub", input, quo), to_i=sym_help.cast_pytorch_to_onnx[dtype])
```

Comment:
Is this `Cast` needed? For `onnx::Sub`, it is enforced that inputs and output all share the same type, so the type of output from `g.op("Sub", input, quo)` must be the same as type of `input`, or else the ONNX graph is invalid.

Reply:
The purpose was to finally cast the output to the same type as input. I fixed this by initially casting 'other' to the same type as input.